x/bank/types/balance.go

```diff
@@ -0,0 +1,93 @@
+package types
+
+import (
+	"bytes"
+	"encoding/json"
+	fmt "fmt"
+	"sort"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/exported"
+)
+
+var _ exported.GenesisBalance = (*Balance)(nil)
+
+// GetAddress returns the account address of the Balance object.
+func (b Balance) GetAddress() sdk.AccAddress {
+	addr, _ := sdk.AccAddressFromBech32(b.Address)
```

Comment:
Why are we ignoring the error from here? Feels like we should be panicking instead.